### PR TITLE
7.9.2 release

### PR DESCRIPTION
**Pull request by nkammah**
This PR prepare `7.9` branch for `7.9.2` release:
- bump chart and Docker images version from `7.9.2-SNAPSHOT` to `7.9.2`
- remove notices on README that this is a development branch and development branch specific install instructions

This is required because released Helm charts will be packaged from this PR commit including the README files that will be available inside the chart packages and in [Helm Hub](https://hub.helm.sh/charts/elastic) pages (example: [Elasticsearch chart page](https://hub.helm.sh/charts/elastic/elasticsearch)).

This change will be reverted in the next PR after release when we will bump `7.9` branch to `7.9.3-SNAPSHOT` version.

### CODE REVIEW

**Review by mgreau — APPROVED** [review]
LGTM

**Comment by nkammah** [comment]
## 7.9.2 - 2020/09/24
* 7.9.2 as the default stack version
* Bump Helm version to 2.16.12 ([@jmlrt](https://github.com/jmlrt))

### Elasticsearch

| PR                                                      | Author                                                 | Title                                                             |
|---------------------------------------------------------|--------------------------------------------------------|-------------------------------------------------------------------|
| [#729](https://github.com/elastic/helm-charts/pull/729) | [@floretan](https://github.com/floretan) | Include pre-releases in the semver range. |
| [#810](https://github.com/elastic/helm-charts/pull/810) | [@luanguimaraesla](https://github.com/luanguimaraesla) | Add loadBalancer externalTrafficPolicy option |
| [#778](https://github.com/elastic/helm-charts/pull/811) | [@jmlrt ](https://github.com/jmlrt) | Fix secrets names in examples |

### Metricbeat

| PR                                                      | Author                                | Title                             |
|---------------------------------------------------------|---------------------------------------|-----------------------------------|
| [#778](https://github.com/elastic/helm-charts/pull/778) | [erihanse](https://github.com/erihanse) | Support secrets |

File: apm-server/README.md [hunk 1]
```diff
@@ -54,15 +52,6 @@ This chart is tested with the latest 7.9.2-SNAPSHOT version.
   - with [Helm 3 (beta)][]: `helm install apm-server elastic/apm-server`
```
Comment (jmlrt) [inline apm-server/README.md:52]:
```suggestion
  - with [Helm 3 (beta)][]: `helm install apm-server --version 7.9.2 elastic/apm-server`
```

Comment (jmlrt) [reply apm-server/README.md:52]: All the installed instructions (Helm2 and Helm3 for all charts) should mention the `--version 7.9.2`

**Review by jmlrt — CHANGES_REQUESTED** [review]
Missing these changes: https://github.com/elastic/helm-charts/pull/803/files#diff-04c6e90faac2675aa89e2176d2eec7d8L106-R117

**Pull request merged**